src/cleanup.js

```diff
@@ -0,0 +1,26 @@
+import { act } from 'react-test-renderer'
+
+let cleanupCallbacks = []
+
+async function cleanup() {
+  await act(async () => {})
+  cleanupCallbacks.forEach((cb) => cb())
+  cleanupCallbacks = []
+}
+
+function addCleanup(callback) {
+  cleanupCallbacks.push(callback)
+}
+
+function removeCleanup(callback) {
+  cleanupCallbacks = cleanupCallbacks.filter((cb) => cb !== callback)
+}
+
+// Automatically registers cleanup in supported testing frameworks
+if (typeof afterEach === 'function' && !process.env.RHTL_SKIP_AUTO_CLEANUP) {
```

Comment:
📓 `process.env.RHTL_SKIP_AUTO_CLEANUP` only needs to be "truthy" and not `true` to work.

Do we want this to explicitly be the string `"true"`?

Reply:
Again, this is implemented as per [`@testing-library/react-testing-library`](https://github.com/testing-library/react-testing-library/blob/master/src/index.js#L9)